posthog/warehouse/api/table.py

```diff
@@ -172,4 +185,11 @@ def update_schema(self, request: request.Request, *args: Any, **kwargs: Any) ->
         table.columns = columns
         table.save()
 
+        # Have to update the `valid` value seperately to the `columns` valye as the columns are required in the `ast.S3Table` class when querying ClickHouse
```

Comment:
typo

Reply:
👍 gonna change in my next PR so that I don't have to go through CI of hell again